specification/cost-management/resource-manager/readme.md

```diff
@@ -406,6 +408,42 @@ directive:
   - suppress: GetCollectionResponseSchema
     from: common-types.json
     reason: 'Operations does not contain a path for individual GET'
+  - suppress: TopLevelResourcesListBySubscription
+    from: costallocation.json
+    reason: 'List by subscription is not supported in cost allocation by desgin'
+  - suppress: PathForResourceAction
+    from: costallocation.json
+    reason: 'This is not a valid scenario for the checkNameAvailability API as the name itself represents an action.'
+  - suppress: PathForPutOperation
+    from: costallocation.json
+    reason: 'Subscripiton and ResourceGroup scope is not supported in cost allocation by desgin'
+  - suppress: RequiredReadOnlySystemData
+    from: costallocation.json
+    reason: 'cost allocation does not return system data and will consider adding it in the future or upcoming api version'
+  - suppress: EnumInsteadOfBoolean
+    from: costallocation.json
+    reason: 'Keeping it as boolean property as per the design'
+  - suppress: NoDuplicatePathsForScopeParameter
+    from: settings.json
+    reason: 'Settings does not use scope for List API'
```

Comment:
>'Settings does not use scope for List API' [](http://example.com/codeflow?start=12&length=42)

I don't think I understand the justification here. This is the list api - does it not use scope?

   "/{scope}/providers/Microsoft.CostManagement/settings": {
      "get": {
        "tags": [
          "Settings"
        ],
        "operationId": "Settings_List",
        "description": "List all cost management settings in the requested scope.",
        "externalDocs": {

Reply:
I don't see what the linter error is complaining about either though TBH. Maybe I'm missing something. @rkmanda @mentat9 see anything I'm missing?